content/en/users/compute/orchestration/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cloud Orchestration
+linkTitle: Cloud Orchestration
+type: docs
+weight: 60
+description: >
+  Compute resource orchestration in the EGI infrastructure
+---
+
+## Overview
+
+One of the challenges for researchers in recent years is to manage an ever
+increasing amount of compute and storage services, which then form ever more
+complex end user applications or platforms.
+
+To address this need, several cloud-based orchestrators are available that can
+support the creation of virtual infrastructures on top of
+Infrastructure-as-a-Service (IaaS) Cloud resources. These tools have different
+levels of abstractions and features. See below to understand which to choose
+(or gets used automatically) in specific scenarios:
+
+<!-- markdownlint-disable line-length -->
+
+| Service Name                                 | Workload Type             | Use-Case |
+| -------------------------------------------- | ------------------------- | -------- |
+| [Infrastructure Manager](im)                 | VMs and containers        | Used to run workloads on a single IaaS Cloud provider. |
+| [Elastic Cloud Compute Cluster](ec3)         | VMs and containers        | Used when you need to run workloads on clusters that can be elastically scaled and potentially span more than one IaaS Cloud provider. |
```

Comment:
Workload Manager missing here. It's probably not in the same category, but still needs to be mentioned

Reply:
No, you are right. II added it to the table, maybe the description is not the best, though.